src/helpers/ethers.ts

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
As far as possible, anything that needs to leverage the ethers dep should happen in this file. As you'll see it isn't a ton of stuff. There are some helpers, some interfaces, and several exports.